src/features/projectsV2/ProjectsMap/index.tsx

```diff
@@ -52,15 +51,12 @@ function ProjectsMap(props: ProjectsMapProps) {
     isMobile: props.isMobile,
     isSearching,
     setIsSearching,
+    page: props.isMobile ? props.page : undefined,
   };
 
   return (
     <>
-      {props.isMobile && (
-        <div className={styles.projectListControlsContainer}>
-          <ProjectListControlForMobile {...projectListControlProps} />
-        </div>
-      )}
+      <MobileControls {...projectListControlProps} />
```

Comment:
I have refactored this component further in one of the sub-branches. Instead of using `<MobileControl>`, we will now use` <MapControls>`, which will serve as a common component for both mobile and desktop.